src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -340,144 +326,125 @@ FindImplicitGemmWrwGTCDynamicXdlopsKernel(const ConvolutionContext& ctx)
                 16  |   0  |1  |0  |0  |0
 
     */
-    int i, j;
-    int max_grid_size  = 0;
-    int cur_grid_size  = 0;
-    int num_cu         = 120;
-    int max_block_size = 0;
+    int max_grid_size                 = 1200;
+    int sel_index                     = -1;
+    int sel_block_size                = 0;
+    int sel_grid_size                 = 0;
+    int sel_log2_gemm_k_global_splits = 0;
+
+    int num_cu                = 120;
+    std::vector<int> nxb_list = {16, 8, 4, 1};
+    std::vector<int> nxe_list = {0, 1};
 
     // i=log2(gemm_m_per_block*gemm_n_per_block)  to find largest kernel
     // switch l and r to get differnet kernel size like 256*64 or 64*256
-    for(i = 15; i > 7; i--)
+    for(int pack = 0; pack < 2; pack++)
```

Comment:
Could you please add a comment here to explain the meaning of `pack` ?

Reply:
pack means two iteration, first normal case, then non-normal case